.editorconfig

```diff
@@ -0,0 +1,28 @@
+root = true
+
+[*]
+indent_style = space
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+[*.py]
+indent_size = 4
+max_line_length = 100
+
+[*.js]
```

Comment:
We don't have `.js`, `.html`, or `.css` files yet. It's foreseeable that the GUI will require some of these formats, so I'm proactively adding them.